### PR TITLE
GEN-2386: Insurance card

### PR DESCRIPTION
**Pull request by julia-andersson**
## [GEN-2386]

-Insurance card updated design

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/hCoreUI/Sources/Views/ContractRow.swift [hunk 1]
```diff
@@ -110,18 +110,15 @@ private struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
...
-                light: hTextColor.Opaque.secondary,
-                dark: hGrayscaleOpaqueColor.greyScale900
-            )
+            hFillColor.Opaque.tertiary
         }
     }
 
     @ViewBuilder var logo: some View {
-        Image(uiImage: hCoreUIAssets.helipadOutlined.image.withRenderingMode(.alwaysTemplate))
+        Image(uiImage: hCoreUIAssets.helipadBig.image.withRenderingMode(.alwaysTemplate))
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/Views/ContractRow.swift:118]: replace this line with
hCoreUIAssets.helipadBig.view
You don't need withRenderingMode(.alwaysTemplate) when using template image

File: Projects/hCoreUI/Sources/Views/ContractRow.swift [hunk 2]
```diff
@@ -165,7 +161,6 @@ private struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
             background
         )
         .border(hBorderColor.primary, width: 0.5)
-        .colorScheme(.light)
         .clipShape(Squircle.default())
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/Views/ContractRow.swift:164]: Switch from Squircle to RounderRectangle. I am not sure about the corner radius amount.

File: Projects/hCoreUI/Sources/Views/ContractRow.swift [hunk 3]
```diff
@@ -179,19 +174,21 @@ private struct StatusPill: View {
...
         }
-        .padding(.vertical, .padding4)
-        .padding(.horizontal, .padding10)
-        .foregroundColor(hTextColor.Opaque.primary).colorScheme(.dark)
-        .background(hTextColor.Translucent.tertiary).colorScheme(.light)
+        .padding(.vertical, 3)
+        .padding(.horizontal, .padding6)
+        .foregroundColor(hTextColor.Opaque.white)
+        .background(hFillColor.Translucent.tertiary).colorScheme(.light)
         .cornerRadius(8)
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/Views/ContractRow.swift:181]: I think 8 should be cornerRadiusXS

**Review by sladan-hedvig — COMMENTED** [review]
Small changes. Nothing huge

File: Projects/hCoreUI/Sources/Views/ContractRow.swift (hunk 3 above)
Comment (julia-andersson) [reply Projects/hCoreUI/Sources/Views/ContractRow.swift:181]: cornerRadiusS

**Review by sladan-hedvig — APPROVED** [review]

**Pull request merged**